src/pages/instances/actions/DeleteInstanceBtn.tsx

```diff
@@ -27,6 +28,7 @@ const DeleteInstanceBtn: FC<Props> = ({ instance }) => {
 
   const handleDelete = () => {
     setLoading(true);
+    setDeleting(true);
```

Comment:
we can replace `isLoading` with `isDeleting` as a prop. no need to maintain state here and inside the parent anymore